x/nft/module/module.go

```diff
@@ -162,12 +162,6 @@ func (AppModule) GenerateGenesisState(simState *module.SimulationState) {
 	simulation.RandomizedGenState(simState)
 }
 
-// ProposalContents returns all the nft content functions used to
-// simulate governance proposals.
-func (am AppModule) ProposalContents(simState module.SimulationState) []simtypes.WeightedProposalContent {
```

Comment:
Forgot this one in https://github.com/cosmos/cosmos-sdk/pull/14728 doing here.
Properly removed in the backported PR 👍🏾